scripts/utils/user_interface_helpers.js

```diff
@@ -13,6 +13,12 @@ const promptUser = function (message) {
   )
 }
 
+function checkNoDuplicatedBracket(argv) {
+  if (new Set(argv.brackets).size !== argv.brackets.length)
+    throw new Error("the parameter --brackets is not supposed to have duplicated entries")
+  return true
+}
+
```

Comment:
This is cool, but I wonder if it could be made more generic like just `checkNoDuplicates` and also be used on other lists that might be provided to our scripts.

Reply:
Also, I would prefer if this function was located in the `default_yargs` file that is more geared toward code that involved the arguments.

Reply:
I'd also like to see a `checkNoDuplicates`, this function could be used in `script/withdraw.js` for the arguments `tokens` and `tokenIds`, for example.